test/WebJobs.Extensions.CosmosDB.Tests/WebJobs.Extensions.CosmosDB.Tests.csproj

```diff
@@ -8,7 +8,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="4.0.3" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.37" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.0" />	  
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="4.0.3" />	  
+    <PackageReference Include="Microsoft.Azure.WebJobs.Host.TestCommon" Version="3.0.30" />	  
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="7.0.0" />
```

Comment:
nit: Is version 7 needed? Shouldn't it be 6 (LTS)?

Reply:
I just picked up the higher major version, anyway is a test project.

Reply:
Yes but 7 is not the LTS. This is a net6 app, not net7, shouldn't we ideally use 6.X?

Reply:
ok, changed to 6.0.0